src/components/common/Input/Input.jsx

```diff
@@ -0,0 +1,13 @@
+export const Input = (props) => {
```

Comment:
в js когда у нас нету типов, лучше вместо просто props делать деструктуризацию пропсов. Так при первомом взглядет будет понятно какие есть пропсы у компонента, иначе мне придется изучать весь код компонента, чтобы понять что у него за пропсы (у большого компонента это займет время).

```suggestion
export const Input = ({ setText, text }) => {
```